projects/hee-shared/src/lib/analytics/analytics.module.ts

```diff
@@ -74,11 +89,13 @@ export class AnalyticsModule {
    * Initaialize window.gtag function for tracking
    */
   private initializeGtag(): void {
-    if (!this.gascript && typeof this.win.gtag === 'undefined') {
+    if (!this.gascript && typeof this.win.gtag === "undefined") {
       this.gascript = true;
       this.win.dataLayer = this.win.dataLayer || [];
-      this.win.gtag = function() { (window as any).dataLayer.push(arguments); }; // The 'arguments' object cannot be referenced in an arrow function in ES3 and ES5. Consider using a standard function expression.
-      this.win.gtag('js', new Date());
+      this.win.gtag = () => {
+        (window as any).dataLayer.push(arguments);
+      }; // The 'arguments' object cannot be referenced in an arrow function in ES3 and ES5. Consider using a standard function expression.
```

Comment:
comment on this line was because I didn't wanna break the google snippet. Not sure if this works / compiles

Reply:
ah yes you are correct. just pushed a fix